src/theory/quantifiers/quantifiers_rewriter.h

```diff
@@ -110,12 +111,11 @@ class QuantifiersRewriter : public TheoryRewriter
                                  Node& var);
   /** get variable elimination
    *
-   * If n asserted with polarity pol entails a literal lit that corresponds
-   * to a variable elimination for some v via the above method, we return true.
-   * In this case, we update args/vars/subs based on eliminating v.
+   * If there exists an n with polarity pol in body, and entails a literal that
+   * corresponds to a variable elimination for some v via the above method, we
+   * return true. In this case, we update args/vars/subs based on eliminating v.
```

Comment:
There is no `pol` defined here. Also not clear what is the method above being mentioned.